lighthouse/classes/services/mongo.py

```diff
@@ -0,0 +1,28 @@
+from flask import current_app as app
+from lighthouse.constants.fields import FIELD_BARCODE, FIELD_LH_SOURCE_PLATE_UUID, FIELD_LH_SAMPLE_UUID
+
+
+class ServiceMongoMixin:
```

Comment:
I would rename this to MongoServiceMixin.

Reply:
Done